### PR TITLE
camelCase RecognizerResult properties

### PR DESCRIPTION
**Pull request by emadelwany**

### CODE REVIEW

File: libraries/Microsoft.Bot.Builder.Ai.LUIS/RecognizerResult.cs [hunk 1]
```diff
@@ -1,12 +1,20 @@
-﻿using Newtonsoft.Json.Linq;
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
 
 namespace Microsoft.Bot.Builder.Ai.LUIS
 {
     public class RecognizerResult
     {
+        [JsonProperty("text")]
```
Comment (cleemullins) [inline libraries/Microsoft.Bot.Builder.Ai.LUIS/RecognizerResult.cs]: This is great. 

There should be a test that enforces these. 

Specifically, create an instance of this, serialize it to JSON, then compare it against a hand-coded and/or known good instance of this JSON doc.

Comment (emadelwany) [reply libraries/Microsoft.Bot.Builder.Ai.LUIS/RecognizerResult.cs]: @cleemullins this test you described would just be testing the JSON.net library, doesn't seem very useful to me?

Comment (emadelwany) [reply libraries/Microsoft.Bot.Builder.Ai.LUIS/RecognizerResult.cs]: Well, I guess it'd also be checking for any typos in the attribute names :)

Comment (emadelwany) [reply libraries/Microsoft.Bot.Builder.Ai.LUIS/RecognizerResult.cs]: @cleemullins I added a unit test

**Pull request merged**